src/main/java/org/example/clients/Client.java

```diff
@@ -0,0 +1,73 @@
+package org.example.clients;
+
+import org.example.server.ServerStrings;
+
+import java.io.*;
+import java.net.Socket;
+import java.net.SocketException;
+
+public abstract class Client {
+    private static Socket Socket;
+    private static BufferedReader in;
+    private static BufferedWriter out;
+
+    public abstract String getGreetings();
+    public abstract String getAnswer();
+    public abstract String getParting();
+    public abstract String getUserName();
```

Comment:
getter for vars that will be declared in classes that will extend Client (this class) is a poor design and against OOP in general.